Frontend/speedcart-react/src/pages/Login/Login.tsx

```diff
@@ -3,8 +3,8 @@ import { useNavigate, useLocation } from 'react-router-dom'; // Necessary for re
 import { GoogleLogin } from '@react-oauth/google';
 
 import { useAuth } from 'shared';
-import { RequestStatus } from '@constants/enums.ts';
-import { AppRoute } from '@constants/routes.ts';
+import { RequestStatus } from '@constants/enums';
```

Comment:
Move these imports down to be under all component imports but above all style imports.